### PR TITLE
Reorg DOT page

### LINKED ISSUE #3840: Reorg DOT page
Minor edits and re-arrangements to the page.

### PR DESCRIPTION
**Pull request by filippoweb3**
This PR focuses on minor re-arrangements.

### CODE REVIEW

File: docs/learn/learn-DOT.md [hunk 1]
```diff
@@ -12,39 +12,42 @@ slug: ../learn-DOT
...
 
+### The Planck Unit
+
 The smallest unit of account in a Substrate network (Polkadot, Kusama, etc.) is the Planck (a
 reference to [Planck Length](https://en.wikipedia.org/wiki/Planck_length), the smallest possible
 distance in the physical Universe). You can compare the Planck to Satoshis or Wei, while the DOT is
-like a bitcoin or an ether. Kusama tokens (KSM) are equal to 1e12 Planck, and Polkadot mainnet DOT
-is equal to 1e10 Planck.
+like a BTC or an ETH. Kusama tokens (KSM) are equal to 10<sup>12</sup> Planck, and Polkadot mainnet
+DOT is equal to 10<sup>10</sup> Planck.
```
Comment (profradha) [inline docs/learn/learn-DOT.md]: Consider replacing the entire paragraph with this text. Could not make suggestions to the lines that were not part of this PR's edits

"The smallest unit for the account balance on Substrate based blockchains (Polkadot, Kusama, etc.) is 
Planck (a reference to [Planck Length](https://en.wikipedia.org/wiki/Planck_length), the smallest 
possible distance in the physical Universe). You can compare DOT's Planck to BTC's Satoshi or ETH's 
Gwei. Polkadot's native token DOT equals to 10<sup>10</sup> Planck and Kusama's native token KSM
equals to 10<sup>12</sup> Planck."

File: docs/learn/learn-DOT.md [hunk 2]
```diff
@@ -12,39 +12,42 @@ slug: ../learn-DOT
...
 
 :::note DOT was redenominated at block #1_248_328
 
-Previously, DOT was denominated as equal to 1e12 Planck, just like Kusama. This denomination is
-deprecated, and, if necessary, referred to as "DOT (old)". See
-[Redenomination of DOT](../general/redenomination.md) for more details.\*
+Previously, DOT was denominated as equal to 10<sup>12</sup> Planck, just like Kusama. This
+denomination is deprecated, and, if necessary, referred to as "DOT (old)". DOT supply was increased
+by 100x and as a consequence now 1 DOT = 10<sup>10</sup> Planck. See
+[Redenomination of DOT](../general/redenomination.md) for more details.
```
Comment (profradha) [inline docs/learn/learn-DOT.md]:
```suggestion
DOT was originally equal to 10<sup>12</sup> Planck just like Kusama (which is referred to as "DOT (old)"), but went through a 
process of [redenomination](../general/redenomination.md) which increased DOT's supply
by 100x. As a consequence, 1 DOT now equals to 10<sup>10</sup> Planck. 
```

File: docs/learn/learn-DOT.md [hunk 3]
```diff
@@ -153,41 +157,41 @@ the future).
...
    [Explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/explorer) page.
 
-## Obtaining Testnet DOT
+## Obtaining Testnet Tokens
 
-DOT are required to make transactions on the Polkadot network. Testnet DOT do not have any value
-beside allowing you to experiment with the network.
+DOT are required to make transactions on the Polkadot network. Testnet tokens exist on copies of the
+Polkadot network (or its parachains) and do not have any value besides allowing you to experiment
+with the network.
```
Comment (profradha) [inline docs/learn/learn-DOT.md]:
```suggestion
DOT are required to make transactions on the Polkadot network. Tokens on Polkadot's Testnets like Westend and Rococo do not have any value besides allowing you to experiment with the features on the network.
```

File: docs/learn/learn-DOT.md [hunk 4]
```diff
@@ -153,41 +157,41 @@ the future).
...
 
-### Getting Westies
+### Getting Tokens on the Westend Testnet
 
-The current testnet is called [Westend](../maintain/maintain-networks.md#westend-test-network) and
-you can obtain its native tokens by posting `!drip <WESTEND_ADDRESS>` in the Matrix chatroom
-[#westend_faucet:matrix.org](https://matrix.to/#/#westend_faucet:matrix.org).
+The current Polkadot testnet is called
```
Comment (profradha) [inline docs/learn/learn-DOT.md]:
```suggestion
Polkadot's testnet is called
```

File: docs/learn/learn-DOT.md [hunk 5]
```diff
@@ -153,41 +157,41 @@ the future).
...
+[Westend](../maintain/maintain-networks.md#westend-test-network) and you can obtain its native
+tokens by posting `!drip <WESTEND_ADDRESS>` in the Matrix chatroom
```
Comment (profradha) [inline docs/learn/learn-DOT.md]:
```suggestion
tokens (called Westies!) by posting `!drip <WESTEND_ADDRESS>` in the Matrix chatroom
```

File: docs/learn/learn-DOT.md [hunk 6]
```diff
@@ -153,41 +157,41 @@ the future).
...
+[#westend_faucet:matrix.org](https://matrix.to/#/#westend_faucet:matrix.org). Your account will be
+credited with 1 WND by default. You can also specify to get more tokens by
+`!drip <WESTEND_ADDRESS> X`, where X is the number of tokens.
 
-You can also make your own WNDs (testnet DOT) by [becoming a validator](learn-validator.md).
+You can make your WNDs (testnet DOT) by [becoming a validator](learn-validator.md). Watch the video
```
Comment (profradha) [inline docs/learn/learn-DOT.md]:
```suggestion
You can also earn WNDs as rewards by [becoming a validator](learn-validator.md) on Westend network. Watch the video
```

**Review by DrW3RK — APPROVED** [review]

File: docs/learn/learn-DOT.md (hunk 1 above)
Comment (filippoweb3) [reply docs/learn/learn-DOT.md]: Thanks, replaced as suggested

**Pull request merged**